db/schema.rb

```diff
@@ -99,8 +98,18 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.boolean "instance_admin", default: false
+    t.string "stripe_customer_id"
```

Comment:
this looks like it's dangling from a different PR

Reply:
I wonder if a pre-checkout hook that rolls back the migrations that have changed between branch X and the branch you're checing out into could exist...